common/amundsen_common/models/feature.py

```diff
@@ -0,0 +1,84 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List, Optional, Dict, Any
+
+import attr
+
+from amundsen_common.models.user import User
+from amundsen_common.models.badge import Badge
+from amundsen_common.models.tag import Tag
+from amundsen_common.models.table import Column, ProgrammaticDescription, Watermark
+from marshmallow3_annotations.ext.attrs import AttrsSchema
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class ColumnItem:
```

Comment:
could we reuse `table.Column` class or it doesn't make sense?

Reply:
this is for the sample response rather than for the partition Column. So basically just the model for the data we need to display a handfull of rows of data for a feature. Notice that partition column does reuse the table column